models/palm_detection_mediapipe/mp_palmdet.py

```diff
@@ -68,17 +69,26 @@ def _postprocess(self, output_blob, original_shape):
         xy2 = (cxy_delta + wh_delta / 2 + self.anchors[:, :2]) * original_shape
         boxes = np.concatenate([xy1, xy2], axis=1)
         # NMS
-        keep_idx = cv.dnn.NMSBoxes(boxes, score, self.score_threshold, self.nms_threshold, top_k=1)
+        keep_idx = cv.dnn.NMSBoxes(boxes, score, self.score_threshold, self.nms_threshold, top_k=self.topK)
         if len(keep_idx) == 0:
-            return None, None, None
-        selected_score = score[keep_idx][0]
-        selected_box = boxes[keep_idx][0]
+            return None
+        selected_score = score[keep_idx]
+        selected_box = boxes[keep_idx]
 
         # get landmarks
-        selected_landmarks = landmark_delta[keep_idx].reshape(7, 2)
-        selected_landmarks = (selected_landmarks / self.input_size + self.anchors[keep_idx]) * original_shape
+        selected_landmarks = landmark_delta[keep_idx].reshape(-1, 7, 2)
+        selected_landmarks = selected_landmarks / self.input_size
+        selected_anchors = self.anchors[keep_idx]
+        for idx, landmark in enumerate(selected_landmarks):
+            landmark += selected_anchors[idx]
+        selected_landmarks *= original_shape
 
-        return (selected_score, selected_box, selected_landmarks)
+        # [
+        #   [bbox_coords, landmarks_coords, score]
+        #   ...
+        #   [bbox_coords, landmarks_coords, score]
+        # ]
+        return np.c_[selected_box.reshape(-1, 4), selected_landmarks.reshape(-1, 14), selected_score.reshape(-1, 1)]
```

Comment:
Need to rerun benchmark on CPU to see whether the speed changes a lot.

Reply:
After testing, there isn‘t much difference in speed on cpu before and after modification. But I found that the speed on RPI-CPU should be about 170ms instead of 89.41ms as written in the README. The time on intel-cpu is about 25ms instead of 15.5. Am I not configured correctly somewhere?

Reply:
Sorry I seem to have mistaken. The results of the test are not very different, do I need to update the benchmark results?

Reply:
no need if minor difference